protocols/kad/src/behaviour.rs

```diff
@@ -2854,14 +2866,21 @@ pub enum QueryInfo {
     },
 
     /// A query initiated by [`Kademlia::get_closest_peers`].
```

Comment:
```suggestion
    /// A (repeated) query initiated by [`Kademlia::get_closest_peers`].
```

To be consistent with the comment updates below.

Reply:
fixed